indico/modules/rb/controllers/user/reservations.py

```diff
@@ -24,6 +24,7 @@
 
 from indico.core.db import db
 from indico.core.errors import IndicoError, NoReportError, NotFoundError
+from indico.modules.rb import settings as rb_settings
```

Comment:
it got super messy since we had `settings` in many modules so we always needed to alias it during import - it was nice within the module but quite ugly anywhere else